src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -314,23 +315,44 @@ def refreshCache(self):
                 possibleLocations = set()
 
                 # all files in job have same location (in se names)
-                rawLocations = loadedJob["input_files"][0]["locations"]
+                #rawLocations = loadedJob["input_files"][0]["locations"]
+                #rawLocations = loadedJob.get("inputDatasetLocations") or []
+                #possibleLocations.update(rawLocations)
+                from pprint import pformat
+                rawLocations = loadedJob.get("inputDatasetLocations") or []
```

Comment:
Yes I think all the code creating inputDatasetLocations and inputDataset need to be reverted and just use

``` python
rawLocations = loadedJob["input_files"][0]["locations"]
```